expected-consensus.md

```diff
@@ -65,22 +65,23 @@ func Mine(minerKey PrivateKey) {
 }
 ```
 
-To check if the miner won the round, she runs `CheckIfWinnerAtRound`. In this method, the miner takes her ticket from a prior round (which round to look at is specified  by the lookback parameter), computes an ElectionProof, and returns whether the proof indicates that the miner has won the round. In the pseudocode below, `IsProofAWinner` is taken from [the mining doc](mining.md#block-validation).
+To check if the miner won the round, she runs `CheckIfWinnerAtRound`. In this method, the miner takes their ticket from a prior round (which round to look at is specified by the randomness lookback parameter), computes an ElectionProof, and returns whether the proof indicates that the miner has won the round. In the pseudocode below, `IsProofAWinner` is taken from [the mining doc](mining.md#block-validation).
 
 ```go
 const RandomnessLookback = 1 // Also referred to as "K" in many places
-const PowerLookback = 1      // Also referred to as "L" in many places
 
 func CheckIfWinnerAtRound(key PrivateKey, n Integer, parentTipset Tipset) (bool, ElectionProof) {
   lbt := ChainTipsMgr.TicketFromRound(parentTipset, n-RandomnessLookback)
 
   eproof := ComputeElectionProof(lbt, key)
 
-  tipset := ChainTipsMgr.TipsetFromRound(n - PowerLookback)
-  minerPower := GetPower(tipset.state, key.Public())
-  totalPower := GetTotalPower(tipset.state)
+  minerPower := GetPower(miner)
+  totalPower := state.GetTotalPower()
 
-  return IsProofAWinner(eproof, minerPower, totalPower), eproof
+  if IsProofAWinner(eproof, minerPower, totalPower)
+    return eproof
+  else
+    return 0
```

Comment:
the signature off the function expects `bool, ElectionProof`, but you return sth different here

Reply:
True. Fixed